src/main/java/com/timeplus/kafkaconnect/TimeplusSinkTask.java

```diff
@@ -133,20 +141,16 @@ private String getCreateRawPayload(String stream) {
         JSONObject payload = new JSONObject();
         payload.put("name", stream);
 
-        JSONObject col = new JSONObject();
-        col.put("name", "raw");
-        col.put("type", "string");
-
         JSONArray cols = new JSONArray();
-        cols.put(col);
+        cols.put(RAW_COL);
         payload.put("columns", cols);
 
         return payload.toString();
     }
 
     private String getCreateJSONPayload(String stream, String event) {
         JSONArray inferredColums = this.infer(event);
-        System.out.println("inferred colume : " + inferredColums);
+        logger.info("inferred columns : " + inferredColums);
```

Comment:
nit: this log seems not super helpful, as one can tell the schema by checking the stream. It would be more useful if the log tells how the columns were actually inferred, rather than what.

Reply:
removed